website/docs/d/eks_node_group_names.html.markdown

```diff
@@ -0,0 +1,45 @@
+---
+subcategory: "EKS"
+layout: "aws"
+page_title: "AWS: aws_eks_node_group_names"
+description: |-
+  Provides a set of node group names for a EKS Cluster
+---
+
+# Data Source: aws_eks_node_group_names
+
+Retrieve information about an EKS Node Group names.
```

Comment:
Something more informative like:
The EKS Node Group Names data source allows access to the list of nodegroups associated with a named EKS cluster.  This will allow you to pass a list of Node Group Names to other resources.